modules/roles_profiles/manifests/roles/gecko_t_osx_1014.pp

```diff
@@ -22,4 +22,5 @@
     include ::roles_profiles::profiles::homebrew
     include ::roles_profiles::profiles::relops_users
     include ::roles_profiles::profiles::gecko_t_osx_1014_generic_worker
+    include ::roles_profiles::profiles::ssh
```

Comment:
Are there any risks in affecting the test pool by turning this on here? Maybe this PR could add the module, and a separate one could make the talos workers use it?

Reply:
I agree; add the module first and then include it in the role after testing.  As it is this module is untested and would definitely have broken puppet in production.